go/worker/sandbox/cgroups/pool.go

```diff
@@ -41,13 +40,11 @@ func NewCgroupPool(name string) (*CgroupPool, error) {
 	if err := syscall.Mkdir(groupPath, 0700); err != nil {
 		return nil, fmt.Errorf("Mkdir %s: %s", groupPath, err)
 	}
-
 	// Make controllers available to child groups
 	rpath := fmt.Sprintf("%s/cgroup.subtree_control", groupPath)
-	if err := ioutil.WriteFile(rpath, []byte("+pids +io +memory +cpu"), os.ModeAppend); err != nil {
-		panic(fmt.Sprintf("Error writing to %s: %v", rpath, err))
+	if err := os.WriteFile(rpath, []byte("+pids +io +memory +cpu"), os.ModeAppend); err != nil {
```

Comment:
There is also a scenario where a user might have systemd delegation configured but they don't have the appropriate controllers made available. Right now this would cause OL to panic here. One way around this would be to first try and probe the system to see which controllers are available and then try to write to only those. If none are available, we fail.